index.html

```diff
@@ -8,15 +8,24 @@
 
   <title>E. Barnes Author</title>
 
-  <meta property="og:title" content="E.M. Barnes Books | Mae: Discovered">
-  <meta property="og:type" content="website">
-  <meta property="og:url" content="https://embarnesbooks.com">
+  <meta property="og:title" content="E.M. Barnes | Mae: Discovered">
+  <meta property="og:type" content="book">
+  <meta property="og:locale" content="en_AU">
+  <meta property="og:url" content="http://embarnesbooks.com">
+  <meta property="og:description" content="The fate of the world rests on her narrow shoulders. Debut novel from E.M. Barnes. Coming soon.">
```

Comment:
Hunting a Queen. The fate of two worlds rests on her narrow shoulders. Debut novel from E.M. Barnes.